test/Microsoft.AspNetCore.SignalR.Tests/project.json

```diff
@@ -11,15 +11,24 @@
     "Microsoft.Extensions.DependencyInjection": "1.2.0-*",
     "Microsoft.Extensions.Logging": "1.2.0-*",
     "Moq": "4.6.36-*",
-    "xunit": "2.2.0-*"
+    "xunit": "2.2.0-*",
+    "Microsoft.AspNetCore.Http.Abstractions": "1.2.0-*",
+    "Microsoft.Extensions.DependencyInjection.Abstractions": "1.2.0-*",
+    "Microsoft.AspNetCore.Hosting.Abstractions": "1.2.0-*",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.2.0-*",
+    "System.Runtime": "4.4.0-*",
+    "Microsoft.Extensions.Logging.Console": "1.2.0-*",
+    "Microsoft.AspNetCore.Server.IntegrationTesting": "0.3.0-*"
   },
   "frameworks": {
     "netcoreapp1.1": {
       "dependencies": {
         "Microsoft.NETCore.App": {
           "version": "1.2.0-*",
           "type": "platform"
-        }
+        },
+        "System.IO.FileSystem": "4.4.0-*",
+        "System.Net.WebSockets.Client": "4.4.0-*"
```

Comment:
I _think_ this should be in the stdlib as well, but I'm much less confident on that one...